pdc/apps/utils/messaging.py

```diff
@@ -124,3 +124,38 @@ def send_message(self, topic, msg):
 
     def listen(self):
         return TestListener(self)
+
+
+class RHMsgMessenger(object):
+    def __init__(self):
+        self.prefix = settings.MESSAGE_BUS['TOPIC_PREFIX']
+        self.config = dict(
+            certificate=settings.MESSAGE_BUS['CERTIFICATE'],
+            private_key=settings.MESSAGE_BUS['CERTIFICATE'],
+            trusted_certificates=settings.MESSAGE_BUS.get('CACERT'),
+            urls=settings.MESSAGE_BUS['URLS'],
+            timeout=settings.MESSAGE_BUS.get('CONNECTION_TIMEOUT'),
+        )
+
+    def _make_headers(self, msg):
+        """
+        Extract all scalar values from the message root and new value and put
+        them as headers.
+        """
+        headers = {}
+        for key, value in msg.items() + msg.get('new_value', {}).items():
+            if isinstance(value, (basestring, int, bool)) or value is None:
+                headers[key] = value
+
+        return headers
+
+    def send_message(self, topic, msg):
+        from rhmsg.activemq.producer import AMQProducer
+
+        headers = self._make_headers(msg)
+        topic = '%s%s' % (self.prefix, topic)
+        logger.info('Sending to %s', topic)
+
+        with AMQProducer(**self.config) as producer:
+            producer.through_topic(topic)
+            producer.send_msg(headers, json.dumps(msg))
```

Comment:
Looking at the `rhmsg` code, this raises an exception if sending fails. Is it OK not to catch it?